### PR TITLE
Make list of repos more configurable

### LINKED ISSUE #803: Remove travis gating from webrender repo.
Removing travis integration altogether would be fine, if that won't break any of the current setup (additional reviewers etc).

### PR DESCRIPTION
**Pull request by jdm**
This fixes #803 and makes it easier for other repositories to be ungated and enable taskcluster support in the future.

---
This change is [Reviewable](https://reviewable.io/reviews/servo/saltfs/821)

### CODE REVIEW

File: homu/files/cfg.toml [hunk 1]
```diff
@@ -94,8 +108,13 @@ secret = "{{ secrets['web-secret'] }}"
     "unicode-script": {},
     "webrender": {
         "extra_reviewers": [ "nical", "jrmuizel", "Gankro" ],
+        "taskcluster": True,
+        "travis": False,
+        "webrender": True,
```
Comment (aneeshusa) [inline homu/files/cfg.toml]: I think this is meant to be `"appveyor": True,`?

Comment (jdm) [reply homu/files/cfg.toml]: Oops. Awkward.

**Pull request merged**